go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/docker/distribution v2.8.1+incompatible
 	github.com/docker/docker v20.10.12+incompatible
 	github.com/go-git/go-billy/v5 v5.3.1
-	github.com/go-git/go-git/v5 v5.4.2
+	github.com/go-git/go-git/v5 v5.4.3-0.20211101214716-934e99b85109
```

Comment:
Which commit is that now? Is it the latest master of go-git?

Reply:
https://github.com/go-git/go-git/commit/934e99b85109874d8a24f344786c30276715bde0

Reply:
are there any improvements in later commits that we would benefit from?

Reply:
Yeah, maybe later. But I think I would prefer not to use too many forks, because dependency updates won't come in then, right?

Reply:
We don't get dependency updates anyway until `go-git` creates new release